src/Collection.php

```diff
@@ -0,0 +1,90 @@
+<?php namespace TightenCo\Jigsaw;
+
+use Illuminate\Support\Collection as BaseCollection;
+
+class Collection extends BaseCollection
+{
+    private $settings;
+    public $name;
+
+    public static function withSettings($settings, $name)
+    {
+        $collection = new static;
+        $collection->settings = $settings;
+        $collection->name = $name;
+
+        return $collection;
+    }
+
+    public function loadItems($items)
+    {
+        $sortedItems = $this->defaultSort($items)->keyBy(function($item) {
+            return $item->filename;
+        });
+        parent::__construct($this->addAdjacentItems($sortedItems));
+
+        return $this;
```

Comment:
I can't figure out what the parent constructor call is for here, what's this meant to be doing?